### PR TITLE
🤖 Simplify `hello-world` stub

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Simplify the `hello-world` exercise's stub. The goal of this is to make things easier for students to get started and to introduce as little syntax as possible.

See https://github.com/exercism/v3-launch/issues/46

### CODE REVIEW

File: exercises/practice/hello-world/hello-world.lisp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 (defpackage #:hello-world
-  (:use #:cl)
+  (:use #:common-lisp)
```
Comment (verdammelt) [inline exercises/practice/hello-world/hello-world.lisp:2]: I'm curious about this change. `COMMON-LISP` correct - but so is `CL` (a nickname for `COMMON-LISP` package. I think (or would like to think) that we've standardized on using `CL` on the track.

Comment (TheLostLambda) [reply exercises/practice/hello-world/hello-world.lisp:2]: My votes goes for `CL` as well! Otherwise this looks good to me!

Comment (ErikSchierboom) [reply exercises/practice/hello-world/hello-world.lisp:2]: I just use the example file as the basis for this PR, so I'll update the example file too.

Comment (verdammelt) [reply exercises/practice/hello-world/hello-world.lisp]: Whoops! I guess I need to grep the track for `(:use #:common-lisp)`... thanks for bringing it to our attention.

**Review by verdammelt — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
I'll wait for @TheLostLambda to approve too.

**Review by TheLostLambda — APPROVED** [review]
Looks good to me!

**Comment by TheLostLambda** [comment]
I think I'll merge this now so I can change the rest of the track to use `CL` too! :slightly_smiling_face: 

Thanks to both @ErikSchierboom and @verdammelt !

**Pull request merged**